template/module/README.rst

```diff
@@ -45,18 +45,18 @@ Known issues / Roadmap
 Bug Tracker
 ===========
 
-Bugs are tracked on `GitHub Issues <https://github.com/OCA/
-{project_repo}/issues>`_.
-In case of trouble, please check there if your issue has already been reported.
-If you spotted it first, help us smashing it by providing a detailed and welcomed feedback `here <https://github.com/OCA/
-{project_repo}/issues/new?body=module:%20
-{module_name}%0Aversion:%20
-{branch}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
-
+Bugs are tracked on `GitHub Issues`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback_.
```

Comment:
I think this keeps the template clearer. I don't think they will forget to copy anything because they will probably copy the full template. That's what it is for.